Index: source/meson.build
===================================================================
--- source.orig/meson.build
+++ source/meson.build
@@ -33,31 +33,9 @@ if xdgautostartdir == ''
   xdgautostartdir = sysconfdir / 'xdg' / 'autostart'
 endif
 
-systemunitdir = get_option('systemunitdir')
-sysusersdir = get_option('sysusersdir')
-tmpfilesdir = get_option('tmpfilesdir')
-if get_option('systemd') and systemd.found()
-  if systemunitdir == ''
-    systemunitdir = systemd.get_variable(pkgconfig: 'systemdsystemunitdir')
-  endif
-  if systemunitdir == ''
-    error('Couldn\'t determine systemd system unit directory.')
-  endif
-
-  if sysusersdir == ''
-    sysusersdir = systemd.get_variable(pkgconfig: 'sysusersdir')
-  endif
-  if sysusersdir == ''
-    error('Couldn\'t determine systemd sysusers directory.')
-  endif
-
-  if tmpfilesdir == ''
-    tmpfilesdir = systemd.get_variable(pkgconfig: 'tmpfilesdir')
-  endif
-  if tmpfilesdir == ''
-    error('Couldn\'t determine systemd tmpfiles directory.')
-  endif
-endif
+systemunitdir = prefix / 'lib/systemd/system'
+sysusersdir = prefix / 'lib/sysusers.d'
+tmpfilesdir = prefix / 'lib/tmpfiles.d'
 
 conf_data = configuration_data()
 conf_data.set_quoted('PROJECT_VERSION', meson.project_version())
@@ -93,8 +71,3 @@ if systemd.found()
     'username': username
   }, section: 'Confiuration')
 endif
-
-meson.add_install_script(
-  'meson_post_install.sh',
-  skip_if_destdir: true
-)
